packages/web3-eth/src/signers/TransactionSigner.js

```diff
@@ -70,13 +71,15 @@ export default class TransactionSigner {
 
         const rlpEncoded = ethTx.serialize().toString('hex');
         const rawTransaction = '0x' + rlpEncoded;
+        const txHash = Hash.keccak256(rawTransaction)
 
         return {
             messageHash: Buffer.from(ethTx.hash(false)).toString('hex'),
             v: '0x' + Buffer.from(ethTx.v).toString('hex'),
             r: '0x' + Buffer.from(ethTx.r).toString('hex'),
             s: '0x' + Buffer.from(ethTx.s).toString('hex'),
-            rawTransaction
+            rawTransaction,
+            txHash,
```

Comment:
same here.

Reply:
Thanks. I removed it.

Reply:
You missed removing comma at this line.

Reply:
Please check again :)

Reply:
Ha I think this is depending on your eslint settings and preference BUT saying that I prefer the trailing comma as when you next add a new property to that object a it’s only 1 git line that changes and not 2! Trailing commas FTW 😁!

Reply:
@joshstevens19 
I prefer dangling comma as well, maybe we should have another PR for adding dangling comma 🙂 . In this PR, I don't want to add trailing comma, since the objective of this PR is not that. :D Thanks!